Source/MMImportCore/ConfigGeneral.cs

```diff
@@ -52,7 +52,7 @@ public static ConfigGeneral Load(string filename)
 
 
 
-        private string outputSchema = "..\\..\\..\\..\\Transforms\\Topo\\OSMMTopography.xsd";
+        private string outputSchema = "Topo\\OSMMTopography.xsd";
```

Comment:
These hard coded paths are pretty bad.  Consider moving this into logic in the constructor that checks the filesystem.